cfme/tests/pod/test_appliance_crud.py

```diff
@@ -13,29 +18,153 @@
 ]
 
 
+ansible_config = """
+# Create an OSEv3 group that contains the masters, nodes, and etcd groups
+[OSEv3:children]
+masters
+nodes
+etcd
+#
+# # Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+# SSH user, this user should allow ssh based auth without requiring a password
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_become must be set to true
+ansible_become=true
+openshift_deployment_type=openshift-enterprise
+
+# uncomment the following to enable htpasswd authentication; defaults to
+#DenyAllPasswordIdentityProvider
+openshift_master_identity_providers=[{{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}}]
+
+openshift_master_default_subdomain="{subdomain}"
+openshift_hosted_registry_routehost="registry.{subdomain}"
+openshift_clock_enabled=true
+
+# cloudforms
+openshift_management_install_management=true
+openshift_management_storage_class=preconfigured
+openshift_management_install_beta=true
+openshift_management_app_template=cfme-template
+openshift_management_project={proj}
+openshift_management_template_parameters={{'FRONTEND_APPLICATION_IMG_NAME': '{app_ui_url}', 'FRONTEND_APPLICATION_IMG_TAG': '{app_ui_tag}', 'BACKEND_APPLICATION_IMG_NAME': '{app_url}', 'BACKEND_APPLICATION_IMG_TAG': '{app_tag}', 'ANSIBLE_IMG_NAME': '{ansible_url}', 'ANSIBLE_IMG_TAG': '{ansible_tag}', 'HTTPD_IMG_NAME': '{httpd_url}', 'HTTPD_IMG_TAG': '{httpd_tag}', 'MEMCACHED_IMG_NAME': '{memcached_url}', 'MEMCACHED_IMG_TAG': '{memcached_tag}', 'POSTGRESQL_IMG_NAME': '{db_url}', 'POSTGRESQL_IMG_TAG': '{db_tag}'}}
+
+# host group for masters
+[masters]
+{host}
+
+# host group for etcd
+[etcd]
+{host}
+
+# host group for nodes, includes region info
+[nodes]
+{host} openshift_node_labels="{{'region': 'infra', 'zone': 'default'}}" openshift_schedulable=true
```

Comment:
Sinlge master/node on one system, cool!

Reply:
hehe, I'd rather prefer to use one master and several nodes but it is what it is